### DISCUSSION TITLE
Welcome to FreeCAD_SheetMetal Discussions!

### DISCUSSION THREAD

**Discussion by shaise** [shaise/FreeCAD_SheetMetal#155] [General]
## 👋 Welcome to SheetMetalWorkbench Discussions.
  Use Discussions to ask questions / feature requests, that are not directly related to bugs. (for bug reports, use the Issues section)
  To get started, leave comment below.

**Comment by lucaspercisi** [comment]
Hello,

I miss a differential feature in fold relief: the creation of a configurable semi-circle with a radius in the middle of the fold with the edge of the metal sheet.

![image](https://user-images.githubusercontent.com/17619128/109239666-b639ec00-77b4-11eb-9c24-9aa314b927ce.png)

This feature is available in creating the fold from a line on the surface and in creating the fold from the length of the metal sheet.

**Comment by bertipbg** [comment]
Hello, I am missing a function, when I create a bend with the "make wall" option, the length is only used in the straight piece after the radius, which is not entirely correct in my opinion, I have been working in an industrial company for 25 years  with sheet metal.  I already asked in the freecad forum, but I was told that the function was intended, because it is said that the bug cannot be dimensioned differently if it has an angle other than 90 °. Normally such bends are dimensioned via the "intersection"  .  So the two outer sides of the part are lengthened until they meet, the point is then dimensioned, it is also possible to lengthen the inner edges and take the measurements from this point of intersection.  Maybe this could be built into a future version, unfortunately my programming skills are not sufficient for this, but theoretically this should not be a big problem.  greetings Martin
![96B33415-5636-4F31-B34F-AE34E626599D](https://user-images.githubusercontent.com/69588845/121815063-a6b3b580-cc74-11eb-9e28-fb73a3a91bed.jpeg)
![F486A57B-A5F4-4BBD-AE1D-9B11A3508555](https://user-images.githubusercontent.com/69588845/121815064-a7e4e280-cc74-11eb-861d-048f5c264d1f.png)

**Reply by bertipbg** [reply]
I hope that I have expressed myself reasonably understandably, my English is not the best.
![90B6B36F-C78E-48C6-8416-AA9A462A6713](https://user-images.githubusercontent.com/69588845/121815474-0743f200-cc77-11eb-82ff-f8a4ca2abb3b.png)

**Comment by wwied** [comment]
first off all - sorry for my english. it's not my native language,
Facing a problem with sheetmetall unfold.
i'm trying to create a round wall part - by bending operation or the use of any other WB`s.
that works fine without problems - including unfolding
if i try to add additional holes into the part - holes on both sides of the bended part - unfolding is failing. I've tried several ways to solve the problem. unfortunately without any result.
any help would be appreciated !
